src/approx_itensornetwork/approx_itensornetwork.jl

```diff
@@ -5,7 +5,7 @@ with the same binary tree structure. `root` is the root vertex of the
 pre-order depth-first-search traversal used to perform the truncations.
 """
 function approx_itensornetwork(
```

Comment:
I forget why this was called `approx_itensornetwork`. Can this just be `contract`? It seems like it is just called from `contract` anyway.